src/rules/noUnboundMethodRule.ts

```diff
@@ -20,22 +20,55 @@ import * as ts from "typescript";
 import * as Lint from "../index";
 
 const OPTION_IGNORE_STATIC = "ignore-static";
+const OPTION_WHITELIST = "whitelist";
+
+const OPTION_WHITELIST_EXAMPLE = [
+    true,
+    {
+        [OPTION_WHITELIST]: ["expect", "typeof"],
+    },
+];
 
 interface Options {
     ignoreStatic: boolean;
+    whitelist: string[];
 }
 
 export class Rule extends Lint.Rules.TypedRule {
     /* tslint:disable:object-literal-sort-keys */
     public static metadata: Lint.IRuleMetadata = {
         ruleName: "no-unbound-method",
         description: "Warns when a method is used outside of a method call.",
-        optionsDescription: `You may optionally pass "${OPTION_IGNORE_STATIC}" to ignore static methods.`,
+        optionsDescription: Lint.Utils.dedent`
+            You may additionally pass "${OPTION_IGNORE_STATIC}" to ignore static methods, or an options object.
+            
+            The object might have two properties:
```

Comment:
> might

Nit: `may`

Reply:
I have changed the description. What does 'Nit' mean? 🤔